CHANGELOG.md

```diff
@@ -4,6 +4,32 @@ All notable changes to this project will be documented in this file.
 Each new release typically also includes the latest modulesync defaults.
 These should not affect the functionality of the module.
 
+## [v5.0.0](https://github.com/voxpupuli/puppet-python/tree/v5.0.0) (2020-09-23)
+
+[Full Changelog](https://github.com/voxpupuli/puppet-python/compare/v4.1.1...v5.0.0)
+
+**Breaking changes:**
+
+- Drop CentOS 6 support [\#566](https://github.com/voxpupuli/puppet-python/pull/566) ([bastelfreak](https://github.com/bastelfreak))
+- Drop EOL Debian 8 [\#563](https://github.com/voxpupuli/puppet-python/pull/563) ([bastelfreak](https://github.com/bastelfreak))
+
+**Fixed bugs:**
+
+- Fix undefined method error in facts [\#555](https://github.com/voxpupuli/puppet-python/pull/555) ([wiebe](https://github.com/wiebe))
+
+**Closed issues:**
+
+- add pip2 to Enum for pip\_provider [\#567](https://github.com/voxpupuli/puppet-python/issues/567)
```

Comment:
this is misleading, as it makes it look we did, as the title suggests

Reply:
added the `skip-changelog` label to that issue and regenerated the changelog